packages/analyticsdx-template-lint/src/linter.ts

```diff
@@ -1233,7 +1236,78 @@ export abstract class TemplateLinter<
     }
   }
 
-  private async lintLayoutCheckNavigationObjects(doc: Document, layoutJson: JsonNode): Promise<void> {
+  private async lintLayoutValidationPages(templateInfo: JsonNode, doc: Document, layoutJson: JsonNode) {
+    // function to read all the templateRequirement tags from the readiness file
+    const readinessTags = caching(async () => {
```

Comment:
This gives us a lazy load of the tags and caches the result, so that way it'll only do the read if there's a validation page with a group with tags.